analytics_data_api/v0/views/learners.py

```diff
@@ -60,6 +60,7 @@ class LearnerView(LastUpdateMixin, CourseViewMixin, generics.RetrieveAPIView):
         Returns metadata and engagement data for the learner in JSON format.
 
             * username: The username of the enrolled learner.
+            * account_url:  URL to learner's account page.
```

Comment:
We had discussed something similar (except doing so on the analytics API) when designing this API. I think it's a reasonable idea, but the problem comes when fetching paginated sets of learners. To list N learners we'd need to send N network requests, which would be very slow.